### PR TITLE
python-3.13: move from rc to release version

### PR DESCRIPTION
**Pull request by ajayk**

### CODE REVIEW

File: python-3.13.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ package:
     memory: 8Gi
   dependencies:
     provides:
-      - python3=0
-      - python-3=0
+      - python3=${{package.full-version}}
+      - python-3=${{package.full-version}}
     runtime:
```
Comment (xnox) [inline python-3.13.yaml:15]: this and the other -dev one will make it the default. Are we ready? has everything completed rebuilding for 3.13?

Comment (smoser) [reply python-3.13.yaml:15]: I doing think we should do this yet.

**Review by smoser — CHANGES_REQUESTED** [review]
Let's land the update change (tag filter, I'm bummed I messed that up) and the version change.

I'm indifferent on the vars/ majorminormicro change. It works either way.

**Review by smoser — COMMENTED** [review]
Can you drop the definitions of the transforms that you have stopped using?

`versionMajMinMic` and `tagname`.

**Review by smoser — APPROVED** [review]
thank you.

**Pull request merged**